doc/facebook_insights.md

```diff
@@ -0,0 +1,50 @@
+# FacebookInsights connector
+
+Import data from facebook insights API.
+
+## Data provider configuration
+
+* `type`: `"MySQL"`
+* `name`: str, required
+* `host`: str, required
+* `user`: str, required
+* `db`: str, required
+* `password`: str
+* `port`: int
+* `charset`: str, default to utf8mb4
+* `connect_timeout`: int
```

Comment:
I think these parameters come from a copy paste and does not relate to Facebook Insights ;)